db/dcrpg/queries.go

```diff
@@ -535,6 +601,31 @@ func RetrieveMissedVotesInBlock(ctx context.Context, db *sql.DB, blockHash strin
 	return
 }
 
+// retrieveAllAgendas returns all the current agendas
+func retrieveAllAgendas(db *sql.DB) (map[string]dbtypes.MileStone, error) {
+	rows, err := db.Query(internal.SelectAllAgendas)
+	if err != nil {
+		return nil, err
+	}
+
+	currentMilestones := make(map[string]dbtypes.MileStone)
+	defer closeRows(rows)
+
+	for rows.Next() {
+		var name string
+		m := dbtypes.MileStone{}
```

Comment:
When doing the zero value init, just do `var m dbtypes.MileStone`, and only when assigning a non-default/zero value, please do `m := `.